Scale_space_reconstruction_3/examples/Scale_space_reconstruction_3/scale_space.cpp

```diff
@@ -36,12 +36,13 @@ int main(int argc, char* argv[]) {
 	reconstruct.reconstruct_surface( points.begin(), points.end(), 4 );
     std::cerr << "Reconstruction done:" << std::endl;
     
+    std::ofstream out ("out.off");
     // Write the reconstruction.
     std::cerr << "Neighborhood radius^2 = " << reconstruct.neighborhood_squared_radius() << std::endl;
     for( std::size_t shell = 0; shell < reconstruct.number_of_shells(); ++shell ) {
-        std::cerr << "Shell " << shell << std::endl;
+      //std::cerr << "Shell " << shell << std::endl;
         for( Triple_iterator it = reconstruct.shell_begin( shell ); it != reconstruct.shell_end( shell ); ++it )
-          std::cout << "3 "<< *it << std::endl; // We write a '3' in front so that it can be assembled into an OFF file
+          out << "3 "<< *it << std::endl; // We write a '3' in front so that it can be assembled into an OFF file
```

Comment:
it happened by accident

On 29/06/2015 10:42, Laurent Rineau wrote:
> In
> Scale_space_reconstruction_3/examples/Scale_space_reconstruction_3/scale_space.cpp
> <https://github.com/CGAL/cgal/pull/141#discussion_r33443947>:
>
>>          for( Triple_iterator it = reconstruct.shell_begin( shell ); it != reconstruct.shell_end( shell ); ++it )
>> -          std::cout << "3 "<< *it << std::endl; // We write a '3' in front so that it can be assembled into an OFF file
>> +          out << "3 "<< *it << std::endl; // We write a '3' in front so that it can be assembled into an OFF file
>
> Est-ce que la modification de
> |Scale_space_reconstruction_3/examples/Scale_space_reconstruction_3/scale_space.cpp|
> est volontaire, ou est-ce une erreur ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CGAL/cgal/pull/141/files#r33443947>.
>